workloads/kube-burner-ocp-wrapper/metrics-profiles/mc-metrics.yml

```diff
@@ -3,12 +3,18 @@
 # Only collecting container and worker nodes CPU/Memory metrics
 
 # Management Node metrics: CPU & Memory
-- query: (sum(irate(node_cpu_seconds_total[2m])) by (mode,instance) and on (instance) label_replace(kube_node_role{role="worker"}, "instance", "$1", "node", "(.+)")) > 0
+- query: (sum(irate(node_cpu_seconds_total[2m])) by (mode,instance) and on (instance) label_replace(cluster:nodes_roles{label_hypershift_openshift_io_control_plane="true"}, "instance", "$1", "node", "(.+)")) > 0
```

Comment:
@mukrishn i believe this label only exists on the non-serving worker.s if we want the serving as well then we need hypershift.openshift.io/request-serving-component="true" too

Reply:
True, this metric is to get the overall worker performance, so I intentionally want to remove the serving component because they are not shared and it won't affect over all performance(wrt 80HC capacity) and also it is per hc data so we cannot aggregate/average it other cluster. 
But to measure serving components, I've added two new metrics and that is going to be per hc - `mgmt-servingNode*`

Reply:
And serving and non-serving might not be of same instance type, so it is better to keep them under different metric. 
Question is do we need to monitor default pool anymore ?